tests/utils/test_sal_on_coco_dets.py

```diff
@@ -36,8 +37,21 @@ def test_warning(self, tmpdir: py.path.local) -> None:
         runner = CliRunner()
 
         if is_usable:
-            with patch('xaitk_saliency.utils.bin.sal_on_coco_dets.is_usable', False):
```

Comment:
Mocking `is_usable` to false doesn't test the section of code that checks if `kwcoco` is installed and then sets `is_usable` to false. I thought it would be more thorough to do it this way so that those lines are covered.